src/_guides/language/language-tour.md

```diff
@@ -3649,17 +3649,20 @@ a possibly time-consuming operation
 (such as I/O),
 without waiting for that operation to complete.
 
-When you need to use a value represented by a Future,
-you have two options:
+The `async` and `await` keywords support asynchronous programming,
+letting you write asynchronous code that
+looks similar to synchronous code.
+
 
-* Use `async` and `await`
-* Use the [Future API](/guides/libraries/library-tour#future)
+<a id="await"></a>
```

Comment:
I'm not seeing any links to this anchor.

If you do want to keep this anchor, maybe move it to L3666 just above the paragraph that starts with the following?

> Code that uses `async` and `await` is asynchronous, ...

Reply:
It's the original ID for this section. I figured I'd leave it as-is, in case anyone had links to it.